libs/providers/ofrep/project.json

```diff
@@ -18,24 +18,14 @@
       ]
     },
     "lint": {
-      "executor": "@nx/linter:eslint",
-      "outputs": ["{options.outputFile}"],
-      "options": {
-        "lintFilePatterns": ["libs/providers/ofrep/**/*.ts", "libs/providers/ofrep/package.json"]
-      }
+      "executor": "@nx/eslint:lint",
+      "outputs": ["{options.outputFile}"]
     },
     "test": {
       "executor": "@nx/jest:jest",
       "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
       "options": {
-        "jestConfig": "libs/providers/ofrep/jest.config.ts",
-        "passWithNoTests": true
-      },
-      "configurations": {
```

Comment:
Did you remove this config or was it done as part of the migration?

Reply:
I did not manually remove it.